### ISSUE TITLE
Feature Request: Neo4j Browser - Add option to collapse child relationships

### ISSUE THREAD

**Issue by billscheidel** [neo4j/neo4j#8797]
Currently when you click on a node in the browser there is an option to expand the children which is incredibly helpful.  This allows you to start with a simple base query (one that I can memorize) and then navigate down relationships as needed.

The problem comes if you expand the wrong node by accident (opening hundreds of nodes in the process) there is no way to undo it.  The screen is now full of unwanted nodes and going to each one and removing them is too time consuming.  You basically have to re-run your query and navigate from scratch.  It would be great if there was an option to collapse immediate child relationships of a node which would be equivalent to visiting each directly connected node and clicking on the 'Remove node from visualization' option.

**Comment by ivanfilippenko** [comment]
I agree.  As a new user of Neo4j, I find the inability to collapse nodes to be the primary Neo4j Browser usability issue so far.  Otherwise, I love many things about the tool, which seems to have come far and continues to evolve.  Please add this feature soon.  Hats off!

**Comment by erNail** [comment]
Definitely needed feature. Especially when you are first discovering your graph and expand a node with hundreds of connections, causing the screen to explode with unwanted nodes. Time to restart your graph journey.

**Comment by djcarter-dv** [comment]
+1

**Comment by LoranHayden** [comment]
eyup, definitely something that would be helpful

**Comment by sshah73** [comment]
Has this feature been added? Would be very helpful to keep visualizing the data without re-running the query.

**Comment by jatincse19** [comment]
is there any way to enable or disable this feature, because this feature has now stopped working in my system.

**Comment by joshuacheong** [comment]
Yes I agree this is a extremely basic and necessary feature of the visualizer - even though I understand this is not the primary way to perform visualizations, it continues to be critical in my development tests. Please let me know if this feature has been included! Thank you!

**Comment by paul-sym** [comment]
+1
